### DISCUSSION TITLE
[Help] 缓存命中率很低正常吗

### DISCUSSION THREAD

**Discussion by BBBoZi** [kkkgo/PaoPaoDNS#11] [问答和帮助]
### 在提交之前，请确认

- [X] 我已经尝试搜索过Issue和看过参数说明，但没有找到相关问题。
- [X] 我正在使用最新的docker镜像版本（可以尝试`docker pull sliamb/paopaodns:latest`后重新创建容器，如果不确定的话可以尝试使用`sliamb/paopaodns:dev`）。

### 脚本自检日志

```txt
我去查看redis中，keyspace_misses很高，缓存命中率很低正常吗？
```

### 问题描述和复现步骤

我去查看redis中，keyspace_misses很高，缓存命中率很低正常吗？

**Comment by kkkgo** [comment]
递归DNS是一个积累的过程，正常的，等你的缓存rdb有几个MB应该就好了。

**Comment by kkkgo** [comment]
除了正常的DNS记录缓存，还有一些主机信息，比如往返时间、无效记录等等，DNS记录在TTL内有调用的话还会进行预读取提前更新，因此基本上不用管unbound的缓存的实现，你可以使用dig命令查询结果在0~10ms的话说明缓存是成功的。

**Comment by BBBoZi** [comment]
> 

好的 明白了 感谢！
